oneview/resource_server_certificate.go

```diff
@@ -0,0 +1,513 @@
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"sha1_fingerprint": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"sha256_fingerprint": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"sha384_fingerprint": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"signature_algorithm": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"state": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"status": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"surname": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"type": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"uri": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"valid_from": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"valid_until": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"version": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+
+			"certificate_status": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"chain_status": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"self_signed": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+						"trusted": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+					},
+				},
+			},
+			"created": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"etag": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"modified": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"state": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"uri": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceServerCertificateCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	servC := ov.ServerCertificate{
+		Category:    d.Get("category").(string),
+		Created:     d.Get("created").(string),
+		Description: utils.NewNstring(d.Get("description").(string)),
+		ETAG:        d.Get("etag").(string),
+		Modified:    d.Get("modified").(string),
+		Name:        d.Get("name").(string),
+		State:       d.Get("state").(string),
+		Status:      d.Get("status").(string),
+		Type:        d.Get("type").(string),
+		URI:         utils.NewNstring(d.Get("uri").(string)),
```

Comment:
URI will be created post creation of this resource, so reading this attribute is not needed.

Reply:
Done